src/System.IO.FileSystem.Watcher/src/Configurations.props

```diff
@@ -2,10 +2,11 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      netcoreapp-FreeBSD;
```

Comment:
Would it be better to just have `-Unix` here? It would take care of all "unknown" Unix flavors, not just FreeBSD.

Reply:
I could be wrong but this also seems to impact final location of the assembly. I had problems before when flipping Linux/BSD build in same build tree. Maybe somebody can comment on.

Reply:
@weshaggard?

Reply:
Yes in theory it should work to target Unix.